Import test cases from DVCS for Device Orientation Event

zqzhang opened the pull request:
https://dvcs.w3.org/hg/geo/rev/5ee2a7297634

Commit:
Import test cases from DVCS for Device Orientation Event

https://dvcs.w3.org/hg/geo/rev/5ee2a7297634

Commit:
Update test cases for DeviceOrientation Event

- Turn device_orientation_api.html and manifest.txt files into mainifest.txt and TODO.txt
- Remove t026.html as it has same checkpoints as screen-upmost.html, and merge values-null.html into screen-upmost.html
- Remove t030.html, t031.html, t032.html as they are incorrect in null attributes checking
- Rewrite other test cases

Commit:
Remove http://w3c-test.org from test files.

Commit:
Address @darobin comments:

- Rename the manual tests as -manual
- Drop the manifest.txt
- Remove all usage of user-scalable=no

zqzhang committed:
Use WebIDLParser.js

Honry commented on orientation-event/free-fall-manual.html:
```diff
@@ -0,0 +1,48 @@
+  </head>
+  <body>
+    <p>Free fall the device to run the test, with the screen horizontal and upmost.</p>
+  	<div id="log"></div>
```
Indent not consistent.

Honry replied:
Please fix all indent issue in other files.

Honry commented on orientation-event/horizontal-surface-manual.html:
```diff
@@ -0,0 +1,36 @@
+        }
+      }, false);
+
+      alert("Put the device on a horizontal surface.");
```
Better to convert this hint to a text tag.

Honry commented on orientation-event/horizontal-surface-manual.html:
```diff
@@ -0,0 +1,36 @@
+          run = true;
+          gamma = e.gamma; // Gamma : angle par rapport a x
+          beta  = e.beta;  // Beta : angle par rapport a y
+          alpha = e.alpha; // Alpha : orientation (N-S-E-O)
```
```alpha``` in this test is unused. Need a assertion for it.

Commit:
Address @Honry comments

- Indent consistently using spaces
- Put user interaction instructions into a paragraph
- Remove unreferenced variable

zqzhang commented:
@Honry thanks for the review. Your comments shall be addressed. Please take another look, thanks.

Commit:
Remove trailing whitespace

Commit:
Remove TODO as tests shall follow the latest spec at

https://w3c.github.io/deviceorientation/spec-source-orientation.html

Honry commented:
LGTM
